docs/api/rate-limits.md

```diff
@@ -47,7 +47,15 @@ The following tables list the rate limits for the NuGet.org API.
 
 ## Package Push and Unlist
 
-| API | Limit Type | Limit Value | API usecase | 
+| API | Limit Type | Limit Value | API use case | 
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 search APIs](search-query-service-resource.md). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interaction.
```

Comment:
Let's link to the overview not search specifically:
https://docs.microsoft.com/en-us/nuget/api/overview